src/game-of-life/time-profiling.md

```diff
@@ -551,10 +559,10 @@ fn live_neighbor_count(&self, row: u32, column: u32) -> u8 {
 }
 ```
 
-Now let's run the benchmarks again!
+Now let's run the benchmarks again! This time output it to `after.txt`.
 
 ```
-$ cargo bench
+$ cargo bench > after.txt
```

Comment:
Nitpick: the below output won't show up if we do `> file` here, so instead we can do `| tee file` to both print the output and capture the output into a file.

Reply:
Ah, of course. Pushing a fix!